### PR TITLE
change the way from escape to encode in 2020.0

### PR DESCRIPTION
**Pull request by cheese8**
PR Refactoring
Describe what this PR does for and how you did.
Fix https://github.com/Tencent/spring-cloud-tencent/issues/247

Does this PR be associated with issue? If so, please adding the issue link below.
change escape & unescape way into encode & decode way

### CODE REVIEW

**Comment by misselvexu** [comment]
Please removed  unused imports , fix ci .

**Comment by misselvexu** [comment]
Recommend combining multiple commits into a single commit .

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/feign/RouterLabelFeignInterceptorTest.java [hunk 1]
```diff
@@ -117,26 +120,26 @@ public void testResolveRouterLabel() {
...
-				for (String value : routerLabels) {
-					Map<String, String> labels = unescape(JacksonUtils.deserialize2Map(value));
-
-					Assert.assertEquals("v1", labels.get("k1"));
-					Assert.assertEquals("v22", labels.get("k2"));
-					Assert.assertEquals("v3", labels.get("k3"));
-					Assert.assertEquals("v4", labels.get("k4"));
-					Assert.assertEquals(headerUidValue, labels.get("${http.header.uid}"));
-					Assert.assertEquals("", labels.get("${http.header.name}"));
+				for (String value : routerLabelsMap.values()) {
```
Comment (SkyeBeFreeman) [inline spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/feign/RouterLabelFeignInterceptorTest.java:133]: this loop doesn't work.

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]

**Comment by cheese8** [comment]
> @cheese8 Recommend combining multiple commits into a single commit .

OK

> @cheese8 Recommend combining multiple commits into a single commit .

that's a good idea.

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/feign/RouterLabelFeignInterceptorTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (cheese8) [reply spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/feign/RouterLabelFeignInterceptorTest.java]: 哈哈，确实是

**Pull request merged**